duden/display.py

```diff
@@ -130,46 +130,49 @@ def describe_word(word):
     print(yellow("=" * len(word.title)))
 
     if word.phonetic:
-        print(white(_("Pronunciation:"), bold=True), word.phonetic)
+        print(white("Pronunciation:", bold=True), word.phonetic)
     if word.alternative_spellings:
         print(
-            white(_("Alternative spellings:"), bold=True),
+            white("Alternative spellings:", bold=True),
             ", ".join(word.alternative_spellings),
         )
     if word.part_of_speech:
-        print(white(_("Word type:"), bold=True), word.part_of_speech)
+        print(white("Word type:", bold=True), word.part_of_speech)
     if word.usage:
-        print(white(_("Usage:"), bold=True), word.usage)
+        print(white("Usage:", bold=True), word.usage)
     if word.frequency:
         commonness = "{label} {frequency}{max_frequency}".format(
-            label=white(_("Commonness:"), bold=True),
+            label=white("Commonness:", bold=True),
             frequency=word.frequency,
             max_frequency=blue("/5"),
         )
         print(commonness)
     if word.word_separation:
         print(
             "{label} {content}".format(
-                label=white(_("Separation:"), bold=True),
+                label=white("Separation:", bold=True),
                 content=str(blue("|")).join(word.word_separation),
             )
         )
 
     if word.origin:
-        print(white(_("Origin:"), bold=True), word.origin)
+        print(white("Origin:", bold=True), word.origin)
 
     if word.grammar_overview:
-        print(white(_("Grammar:"), bold=True), word.grammar_overview)
+        print(white("Grammar:", bold=True), word.grammar_overview)
+
+    if word.word_pronunciation_audio:
+        print('{label} {content}'.format(label=white('Pronunciation audio:', bold=True), content=white(word.word_pronunciation_audio, bold=False)))
```

Comment:
This line is too long. This project uses a maximum of 88 characters (as seen in configuration file:)
```
$ git grep 'line-length'
pyproject.toml:line-length = 88
```

An easiest way to fix this is to run the `make autoformat` Makefile task which autoformats the code into a standard form.

Reply:
Understood. I fixed it.